grid2op/Reward/_alertCostScore.py

```diff
@@ -0,0 +1,79 @@
+# Copyright (c) 2023, RTE (https://www.rte-france.com)
+# See AUTHORS.txt
+# This Source Code Form is subject to the terms of the Mozilla Public License, version 2.0.
+# If a copy of the Mozilla Public License, version 2.0 was not distributed with this file,
+# you can obtain one at http://mozilla.org/MPL/2.0/.
+# SPDX-License-Identifier: MPL-2.0
+# This file is part of Grid2Op, Grid2Op a testbed platform to model sequential decision making in power systems.
+
+import numpy as np
+from grid2op.Reward.baseReward import BaseReward
+from grid2op.Reward._newRenewableSourcesUsageScore import _NewRenewableSourcesUsageScore
+from grid2op.dtypes import dt_float
+from grid2op.Exceptions import Grid2OpException
+        
+class _AlertCostScore(BaseReward):
+    """
+
+    INTERNAL
+
+    .. warning:: /!\\\\ Internal, do not use unless you know what you are doing /!\\\\
+            It **must not** serve as a reward. This scored needs to be **MAXIMIZED**,
+            as it is a negative! Also, this "reward" is not scaled or anything. Use it as your
+            own risk.
+
+    Implemented as a reward to make it easier to use in the context of the L2RPN competitions, this reward is based on the "alert feature" 
+    where the agent is asked to send information about potential line overload issue on the grid after unpredictable powerline
+    disconnection (attack of the opponent).
+    The alerts are assessed once per attack. In this scheme, this "reward" computed the assistant"cost score", which penalized the number of alerts
+    the assistant have produced during an episode. It should not be used to train an agent.
+    
+    """
+    def __init__(self, logger=None):
+        BaseReward.__init__(self, logger=logger)
+        self.reward_min = dt_float(-1.0)
+        self.reward_max = dt_float(1.0)
+        self._is_simul_env = False
+        self.total_nb_alertes_possible = None
+        self.total_nb_alerts = None
+        
+    def __initialize__(self, env):
```

Comment:
This will not be called, in grid2op it's `def initialize(self, env):` (see doc of BaseReward)